EIPS/EIP-draft-getLogs-by-block-hash.md

```diff
@@ -0,0 +1,50 @@
+## Preamble
+```
+EIP: <to be assigned>
```

Comment:
Take number 234.  Also move this file to EIPS/eip-234.md

Reply:
Fixed.